package.json

```diff
@@ -273,7 +273,7 @@
     "@types/react-dom": "^16.0.7",
     "@types/react-redux": "6.0.4",
     "awesome-typescript-loader": "^5.2.0",
-    "brave-ui": "github:brave/brave-ui#18386d1f0c65cbb5ad624fa48a761f0f2c58dbd9",
+    "brave-ui": "github:brave/brave-ui#1640d490df748eaf10246f95f546a300bbbe2be3",
```

Comment:
Note: this hash is from an unmerged ui PR, once this PR has been approved, it will be merged in to brave-ui master then this will get updated.